digits/templates/models/images/classification/new.html

```diff
@@ -376,14 +382,16 @@ <h4 style="display:inline-block;">Python Layers</h4>
               )
         .done(function(data) {
                 data = $.parseJSON(data);
-                $('textarea[name=custom_network]').val(data['network']);
+                editor.setValue(data['network']);
+                editor.setValue(editor.getValue(), 1); // Moves cursor to end of editor
```

Comment:
Can we move the cursor to the top instead of the end? Did you have a particular reason for wanting to see the end first?